index.js

```diff
@@ -200,7 +198,7 @@ SteamCommunity.prototype.login = function(details, callback) {
 
 				self.setCookies(cookies);
 
-				callback(null, sessionID, cookies, steamguard, disableMobile ? null : oAuth.oauth_token);
+				callback(null, sessionID, cookies, steamguard, !disableMobile && body.oauth ? oAuth.oauth_token : null, body);
```

Comment:
body added to callback to enable debugging it in cases when steam changes the response